### ISSUE TITLE
Cross Browser Extension Support

### ISSUE THREAD

**Issue by koustuvsinha** [vickychijwani/quickwiki#9]
Currently we are only focusing on Chrome, but I think to get greater user base we should now focus on cross browser support.
Some folder structure changes are also needed. Please look into [this tutorial](http://frontendbabel.info/articles/developing-cross-browser-extensions/) for details

**Comment by vickychijwani** [comment]
It should be much easier to expand our Chrome user-base itself as compared to adding support for other browsers. If we can't expand on Chrome, we'll face the same issue on other browsers, and besides, Chrome's user-base is the largest and its extensions have far more reach because of their ease of installation and use (I'm a Firefox user myself, and it's not as easy to find and use FF addons).

I think we're lacking in publicity and polish more than anything else. The user-base has hovered in the 250-500 range for a several years now. If we invest some time in spreading the word online and listening to user feedback, we should be able to expand well. What do you think?

**Comment by koustuvsinha** [comment]
Yes, building our userbase in Chrome is the first step. We can start spreading the word through social networking I suppose.
Once we have a considerable userbase (> 1000) then we can look into FF support
